File: be/authentication/src/main/java/com/example/authentication/model/dto/UserDto.java
package com.example.authentication.model.dto;

import lombok.*;

import java.util.ArrayList;
import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class UserDto {
    private Integer id;
    private String username;
    private String password;
    private String phonenumber;
    private List<RoleDto> roles = new ArrayList<>();
    private Integer version;

    @Override
    public String toString() {
        return "UserDto{" +
                "id=" + id +
                ", userName='" + username + '\'' +
                ", password='" + password + '\'' +
                ", version=" + version +
                '}';
    }


}

File: fe/Driver/app/src/main/java/com/example/driver/data/dto/UserDto.kt
package com.example.driver.data.dto

import com.google.gson.annotations.SerializedName

data class UserDto(
//    var authorities: String,
    var password: String?,
    var username: String?,
    @SerializedName("phonenumber")
    var phoneNumber: String?,
    @SerializedName("otp", alternate = ["onceTimePassword"])
    var otp: String? = null

)